generators/cypress/templates/src/test/javascript/cypress/tsconfig.json.ejs

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
you can replace all by: 
```
{
  "references": [
    {
      "path": "./../../../../tsconfig.json"
    }
  ]
}
```

Reply:
[jhipster#12310](https://github.com/jhipster/generator-jhipster/pull/12310)